src/execution/ValueSetHelper.ts

```diff
@@ -112,8 +113,12 @@ export function getMissingDependentValuesets(
   // create an array of valueset urls
   const vsURLs: string[] = libraryEntries.reduce((acc, lib) => {
     const libraryResource = lib.resource as fhir4.Library;
-    if (!libraryResource || !libraryResource.dataRequirement || !(libraryResource.dataRequirement.length > 0)) {
-      throw new Error('Expected library entry to have resource with dataRequirements that have codeFilters');
+    if (!libraryResource) {
+      throw new UnexpectedResource('Library entry not included in measure bundle');
+    } else if (!libraryResource.dataRequirement || !(libraryResource.dataRequirement.length > 0)) {
+      throw new UnexpectedProperty(
+        'Expected library entry to have resource with dataRequirements that have codeFilters'
+      );
```

Comment:
Here's what's causing the reduced coverage. Can try to remedy this if people think its important, but my feeling is that it's a very direct translation from the previous lines, and doesn't change the functionality